targets/TARGET_Maxim/TARGET_MAX32630/gpio_irq_api.c

```diff
@@ -118,7 +118,11 @@ int gpio_irq_init(gpio_irq_t *obj, PinName name, gpio_irq_handler handler, uint3
 
     /* enable the requested interrupt */
     MXC_GPIO->inten[port] |= (1 << pin);
-    NVIC_EnableIRQ((IRQn_Type)((uint32_t)GPIO_P0_IRQn + port));
+	if (port >= 7) {
+		NVIC_EnableIRQ((IRQn_Type)((uint32_t)GPIO_P7_IRQn + (port - 7)));
+	} else {
+		NVIC_EnableIRQ((IRQn_Type)((uint32_t)GPIO_P0_IRQn + port));
+	}
```

Comment:
You have changed how the ports are used here. How have you verified that this works as expected now?

Reply:
This is my first PR. I verified that it works in my local PC. Do I have to follow a testing procedure like greentea or unittest for this type of changes?